### PR TITLE
Prepend chat room channel name

### LINKED ISSUE #148: Add gitterbot to remaining FCC rooms
Gitterbot is awesome. Campers are using it like crazy. Have you been monitoring the statistics?
He does a few key things remarkably well
I was wondering whether you could roll him out in the rest of our rooms
There are about 400 of these - is there a good way to do these in batch? They’re listed here: https://github.com/FreeCodeCamp/freecodecamp/wiki/Official-Free-Code-Camp-Chat-Rooms and https://github.com/FreeCodeCamp/freecodecamp/wiki/List-of-Free-Code-Camp-city-based-Campsites

### PR DESCRIPTION
**Pull request by abhisekp**
Refactors and prepend channel names to room names.

Close #148 and close #153 

cc: @dcsan

### CODE REVIEW

File: nap/data/RoomData.js [hunk 1]
```diff
@@ -128,479 +128,457 @@ var bonfireDashedNames = [
...
+    "FreeCodeCamp/Hardware",
+    "FreeCodeCamp/Music",
+    "FreeCodeCamp/Photography",
+    "FreeCodeCamp/Saving",
+    "FreeCodeCamp/SelfImprovement",
+    "FreeCodeCamp/Sports",
+    "FreeCodeCamp/TVandMovies",
     "dcsan/botzy",
     "dcsan/gitterbot"
 ];
```
Comment (abhisekp) [inline nap/data/RoomData.js]: New chat rooms can be added here.

**Comment by abhisekp** [comment]
@dcsan please look into this and merge if this adheres to the requirements you asked.

**Comment by dcsan** [comment]
I think this looks good, but your linter settings are different to mine on space before paren on func name - so the whole code looks different. in general that's bad "PR etiquette" as i have to now check every function signature...

-    getAnnounceMessage: function(opts) {		+    getAnnounceMessage: function (opts) {

**Comment by abhisekp** [comment]
@dcsan the default is `"no-spaced-func": 0` i.e. don't show any error if there is any space between function name and parenthesis.
Moreover, a linter is not responsible for formatting. One must use **js-beautify** for taking care of the source formatting. It was not in the repo. So I had proposed one and you've already merged it.
You've already merged a `.jsbeautifyrc` file in https://github.com/dcsan/gitterbot/blob/master/nap/.jsbeautifyrc
**Repo .eslintrc:** https://github.com/dcsan/gitterbot/blob/master/nap/.eslintrc
**Default .eslintrc:** https://github.com/eslint/eslint/blob/master/conf/eslint.json

**Comment by abhisekp** [comment]
@dcsan though the issue is, In the `.jsbeautifyrc` file, the statement `"space_after_anon_function": false` clearly mentions not to keep any spaces between the `function` keyword and parenthesis in anonymous functions. 
**See:** https://github.com/dcsan/gitterbot/blob/master/nap/.jsbeautifyrc#L10

So I must fix my editor.

**Comment by abhisekp** [comment]
City chat room needs to be removed. And source code needs to be formatted to it's default format.

**Comment by abhisekp** [comment]
This PR is redundant after removal of city rooms and merging #190

**Pull request closed without merging**